pages/common/mdp.md

```diff
@@ -0,0 +1,19 @@
+# mdp
+
+> A command-line based tool to make presentations from markdown formatted files.
+
+- Start a presentation from a markdown file:
```

Comment:
Does this make the presentation in the terminal? if so, that should be explicit in the description. "Start" is kinda vague.